config/routes.rb

```diff
@@ -25,6 +29,17 @@
   
   resources :payments, only: [:new, :create]
   
+  namespace :api do
+    controller :deals do
+      get 'deals/live' => :live
+      get 'deals/past' => :past
+    end
+
+    controller :users do
+      get 'my_orders' => :my_orders
```

Comment:
If a route is like `/api/users/orders` makes more sense than `/api/users/my_orders`

```suggestion
      get 'orders' => :orders
```

Reply:
its mentioned in the docs, so that's why used it
![image](https://github.com/Ratchet1999/FlashSale/assets/62827625/8cbe7810-5e15-4c97-87a9-e27e05f51b5b)

Reply:
its mentioned in the docs so that's why used it 
![image](https://github.com/Ratchet1999/FlashSale/assets/62827625/e2b188a1-4d1e-4340-b016-b1c2c2d29138)

Reply:
ok